index.html

```diff
@@ -0,0 +1,291 @@
+                  class="responsive sectiontwo__container__items__item__responsive__respo"
+                />
+              </div>
+              <span>Lecture</span>
+              <p>
+                Listen to the speakers from various countries about the messages
+                of sharing and opening
+              </p>
+            </div>
+            <div class="sectiontwo__container__items__item">
+              <div class="sectiontwo__container__items__item__responsive">
+                <img
+                  src="static/imgs/forum.svg"
+                  alt="forum"
+                  class="responsive sectiontwo__container__items__item__responsive__respo"
+                />
+              </div>
+              <span>CC Exhibition</span>
+              <p>
+                Appreciate various creations applying CC license of participants
+                from this community
+              </p>
+            </div>
+            <div class="sectiontwo__container__items__item">
+              <div class="sectiontwo__container__items__item__responsive">
+                <img
+                  src="static/imgs/ccIgnite.svg"
+                  alt="ccIgnite"
+                  class="responsive sectiontwo__container__items__item__responsive__respo"
+                />
+              </div>
+              <span>Forum</span>
+              <p>
+                Have the time to share your thoughts and opinionswith experts
+                for each topic
+              </p>
+            </div>
+            <div class="sectiontwo__container__items__item">
+              <div class="sectiontwo__container__items__item__responsive">
+                <img
+                  src="static/imgs/workshop.svg"
+                  alt="workshop"
+                  class="responsive sectiontwo__container__items__item__responsive__respo"
+                />
+              </div>
+              <span>Workshop</span>
+              <p>
+                Denotaing and small community using open sources and license
+                than sitting
+              </p>
+            </div>
+            <div class="sectiontwo__container__items__item">
+              <div class="sectiontwo__container__items__item__responsive">
+                <img
+                  src="static/imgs/workshop.svg"
+                  alt="workshop"
+                  class="responsive sectiontwo__container__items__item__responsive__respo"
+                />
+              </div>
+              <span>CC Ignite </span>
+              <p>
+                Get opportunity to network the CC affiliates around the world,
+                also after conference
+              </p>
+            </div>
+          </div>
+          <a class="sectiontwo__container__title display" href="#"
+            >SEE THE WHOLE PROGRAM</a
+          >
+          <a class="sectiontwo__container__title" href="#" id="second">
+            Join unicef's annual human support</a
+          >
```

Comment:
- [ ] Kindly note that this button should look more of a link on desktop view and more of a button in mobile view.
- [ ] The button should also be centered so that it matches the Behance design.

**This is how I am seeing it right now:**

![image](https://user-images.githubusercontent.com/86318284/197013429-6b870662-7c5c-44b0-9750-3bc745390edb.png)

**This is the desired design:**

![image](https://user-images.githubusercontent.com/86318284/197013596-19506206-bb51-4b86-8fe2-a0d3e9300c85.png)

Reply:
- [ ] As previous reviewer suggested,  you button should be centered and the text inside should be aligned center as well. This will make you website look similar to design provided from behance 
![image](https://user-images.githubusercontent.com/43877689/197211799-2db7d5b2-2a5a-44b4-8ef5-b662673291c0.png)